### ISSUE TITLE
Suggestions for later round play for RP-Based Roles

### ISSUE THREAD

**Issue by El-Fishii** [Tk420634/ARFS-2#191]
**What I feel is a "problem"**
So as I understand we'll be getting secret soon from higher pop's that'll happen. 
Now while this will give nearly every role something to do it's still "not enough" as I would think.
A Botanist to have something to do would need Science or a Chef.
A miner needs Science to be useful. A HoP needs... well miners and crew to boss around. while Captain could just... do everything himself.
A bartender needs science for some drinks) or anyone to bring him chemicals.) but mainly needs RP interaction with other crew members. But if everyone is focusing on their job and not visiting but for running to the food counter and grabbing meat bread and running off. He's relatively useless.
**Now** while that helps my side I understand the other side. Secret gives Security plenty to do with the chaos they'll be doing. If they break things engineers have something to do. But they're not really my point of focus for this. My focus are the "assistant" based jobs that people play mostly for ERP or flavor.
-Chaplain
-Bartender
-Chef
-Librarian
-Janitor
Not really a problem, just something to do to keep people from thinking they need to do important jobs and not get bored.

**What i'm suggesting**
*Disclaimer*: Now I understand I don't know coding but I am willing to learn if someone would teach me so I can help. So what I am suggesting I don't understand the full work it would take to implement.

Objectives. Give them RP based objectives so they have something to work for during the round while security is hunting down the antags etc. 
Ex 1. The chaplain can have an objective to convert a certain number of people to his religion. To have someone confess to their sins and make up for them. Maybe someone is having a hard time and he's to help guide them. Besides obj's if they're in a Secret round needing actual play.
Ex 2. A librarian could be tasked with having a pizza party in the library and inviting people over to play a game of tabletop. Checkout a X number of books to people. 
Maybe they've flavoured themselves as a journalist instead of a librarian. then they could get a objective to cover a juicy story, maybe on what's happening with the actual antags on the station, to get why they're doing what they do. Etc. 

There can be other things to do but this is just a suggestion to make it so people don't feel like they need to play bigger roles that they don't know how to play to have fun. Having something to do is better than not. When you join as an engineer at round start you GET TO WORK. Engine, Atmos, substations. Maybe fixing some stuff if the alarms went out. amoungst countless other bullshit that'll break some part of the ship that gives you something to do. But a Chaplain? You join. Get your items. And wander around trying to talk to people who have too much to do to talk to you. you just sit there with nothing to do and eventually go brain dead.

**TL'DR**
Basically I'm asking for objectives to be added to RP based roles so that people have "something to work for" instead of wandering mindlessly until they get bored and leave. You could update objectives at every crew transfer vote so that if they've completed them they get more etc. Just to keep non-essential roles still being played by people so there is actual RP engagement and it doesn't transform into this convoluted fuckfest of work and getting stuff done with no RP happening.

-Fishii
